contracts/rootchain/RootChain.sol

```diff
@@ -314,6 +315,7 @@ contract RootChain is Ownable {
         view
         returns (address, uint256, uint256[3], uint256)
     {
+        emit FinalizedExit(priority, exits[priority].owner, exits[priority].amount);
```

Comment:
Why are we emitting a FinalizedExit here, this is just a function to get an exit.

Reply:
same as above